### DISCUSSION TITLE
Feedback and nitpicking

### DISCUSSION THREAD

**Discussion by HeinzNepe** [victorbalssa/abacus#129] [General]
First off, I would like to mentionlove the abacus app. Makes  it much more convenient to use firefly. 

Here is my feedback for the app; and some quick thoughts on what could be better

In the newest update (version 9.0), the placeholder value for the amount in transactions is in dollars. Would it be possible to change it away from dollars, as my firefly is configured with another currentcy?

When trying to submit the transactions I create, I run in to the issue of not being able to click "submit" as it is behind my keyboard. 

Thanks for the amazing work you do! This is all just nitpicking, I've got to say I believe the app has come far from its initial release.  

Best regards
HeinzNepe

**Comment by victorbalssa** [comment]
Hey @HeinzNepe,

Thank you so much! I love reading comments like this.

I will address everything you mentioned in the next patch `0.9.1`.

- Amount placeholders will be back as blank `0.00` without currency if no source account selected.
- I will add some space to the end of the transaction form.

Stay tuned!
